README.md

```diff
@@ -206,12 +206,12 @@ var publicKey = fs.readFileSync('/path/to/public.pub');
 app.use(jwt({ secret: publicKey }));
 ```
 
-RSA algorithm is also supported by providing jwks-rsa.expressJwtSecret callback in the
+RSA algorithm is also supported by providing jwks promise provider in the
```

Comment:
I've changed middle paragraph a little bit to correspond to previous commit:

The signature of this function should be (header) => [Promise(secret)], where
header is token header. For instance to support JWKS token header should contain
alg and kid: algorithm and key id fields respectively.